flycheck.el

```diff
@@ -834,6 +834,7 @@ This variable is a normal hook.  See Info node `(elisp)Hooks'."
     (define-key map "h"         #'flycheck-display-error-at-point)
     (define-key map "H"         #'display-local-help)
     (define-key map "i"         #'flycheck-manual)
+    (define-key map "r"         #'flycheck-explain-error-at-point)
```

Comment:
We can use `x` here and drop the original binding for `C-c ! x`, in my opinion.   I think explain is much more useful so should get a better binding.